portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,135 @@
+		<div class="row">
+			<div class="col">
+                <!-- About Sam section -->
+				<h3 class="feature-title">About Me</h3>
+				<img src="images/column-1.jpg" class="img-fluid">
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque interdum quam odio, quis placerat ante luctus eu. Sed aliquet dolor id sapien rutrum, id vulputate quam iaculis.</p>
+                <!-- Fun Facts button -->
+                <button id="project-button" onclick="funFact()"> Fun Facts! </button>
+                <div id="fact-container"></div>
+
+            </div>
+			<div class="col">
+
+                <!-- Skills -->
+				<h3 class="feature-title">Skills</h3>
+				<img src="images/column-2.jpg" class="img-fluid">
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque interdum quam odio, quis placerat ante luctus eu. Sed aliquet dolor id sapien rutrum, id vulputate quam iaculis.</p>
+			</div>
+			
+		</div> 
+	</div>
+
+    <!-- Projects Section -->
+    <div id= "projects" class="project-area">
+		<div class="column">
+            <br>
+            <h3 class="project-title">Projects </h3>
+            <br>
+            <h3 class="project-name">Personal Website</h3>
+            <p class="project-p">Developed a web platform to display my web projects and information about myself.</p>
+            <br>
+            <br>
+            <h3 class="project-name">Orgspace</h3>
+            <p class="project-p">Developed a web platform for non-profit organizations to connect with other organizations and individuals at Google CSSI in Mountain View in 2019.</p>
+            <br>
+            <div class="row">
+            </div>
+        </div>
     </div>
+
+	
+	<footer class="page-footer">
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-8 col-md-8 col-sm-12">
+					<h6 class="text-uppercase font-weight-bold">Additional Information</h6>
+					<p>If you would like to connect with me soon, please reach out with the contact information I have displayed on the right.</p>
+					<a href="https://www.linkedin.com/in/samuelsandoval1/"> <img class="foot" src="images/linkedin.png" alt="LinkedIn"/> </a>
+                    <a href="https://github.com/samuelsandoval1/"> <img class="foot" src="images/github.png" alt="Github"/> </a>
+
+				</div>
+				<div class="col-lg-4 col-md-4 col-sm-12">
+					<h6 class="text-uppercase font-weight-bold">Contact</h6>
+					<p>1170 Bordeaux Dr, Sunnyvale, CA </p>
+					<br/>samuelsandoval@google.com
+					<br/>+ 01 234 567 88
+					<br/>+ 01 234 567 89</p>
+				</div>
+			</div>
+		</div>
+		<div class="footer-copyright text-center">© 2020 Copyright: samuelsandoval-step-2020.appspot.com</div>
+	</footer>
+
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+      crossorigin="anonymous"
+    ></script>
+    <script src="bootstrap/js/bootstrap.min.js"></script>
```

Comment:
Do you know if both of these are needed?

Reply:
Yes they layout the header

Reply:
Sorry I didn't specify which files I was referring to. 

My main concern is whether the local bootstrap.min.js is the same as the bootstrap.min.js from the Bootstrap CDN. If they are the same, consider just using one of them.

Reply:
Oh ok

Reply:
this is a local bootstrap js file i have